### PR TITLE
Newsweek-clone

### PR DESCRIPTION
**Pull request by abongsjoel**
In this project, I clone the Newsweek homepage using semantic HTML5 elements, Bootstrap4 as well as plain CSS3. The page starts with a header section that uses the Bootstrap4 navbar classes to make it responsive. The main part of the page contains a set of articles that are aligned differently on different screen sizes using Bootstrap4 Grid system.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,335 @@
+/*** General ***/
+body {
+  background-color: #f4f5f6;
+}
+
+.pt-7 {
+  padding-top: 4rem !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove the `important`

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+}
+
+@media only screen and (min-width: 992px) {
+  .pt-7 {
+    padding-top: 10.5rem !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove the `important`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+  font-weight: bold;
+}
+
+@media only screen and (min-width: 768px) {
+  h3 {
+    font-size: 1.3rem !important;
+  }
+
+  h4 {
+    font-size: 1.2rem !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove the `important`

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+}
+
+header .nav-link {
+  font-size: 1.25rem;
+  padding-top: 0 !important;
+}
+
+@media only screen and (min-width: 992px) {
+  header .nav-link {
+    padding: 0.5rem 0 !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove the `important`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1239 @@
...
+              </figcaption>
+            </figure>
+            <figure class="col my-1">
+              <img class="w-100" src="https://d.newsweek.com/en/full/1669014/canada-emigraton-trump-presidential-election.jpg?w=397&h=397&f=01273440ad3df483a50a5546ffd0a938" alt="">
+              <figcaption>
+                <h4 class="text-secondary mt-3">'We Moved To Canada in 2016 After Trump Won, We Won't Be Returning''</h4>
+              </figcaption>
+            </figure>
+            <figure class="col my-1">
+              <img class="w-100" src="https://d.newsweek.com/en/full/1667965/chess-womens-chess-queens-gambit.png?w=397&h=397&f=35386f73ca5a1a7a87a128f9bd61dd2c" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add the description for `alt` attribute

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1239 @@
...
+          <h2 class="text-muted">MY TURN</h2>
+          <div class="row">
+            <figure class="col my-1">
+              <img class="w-100" src="https://d.newsweek.com/en/full/1669118/car-accident-teenager-utah.jpg?w=397&h=397&f=45dbdba3cb959860cb9573a9471cd74a" alt="">
+              <figcaption>
+                <h4 class="text-secondary mt-3">'I'm 17. I Lost Both My Legs After a Head-On Car Crash'</h4>
+              </figcaption>
+            </figure>
+            <figure class="col my-1">
+              <img class="w-100" src="https://d.newsweek.com/en/full/1669014/canada-emigraton-trump-presidential-election.jpg?w=397&h=397&f=01273440ad3df483a50a5546ffd0a938" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add the description for `alt` attribute

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1239 @@
...
+          </figcaption>
+        </figure>
+        <figure class="my-4">
+          <img class="w-100" src="https://d.newsweek.com/en/full/1669014/canada-emigraton-trump-presidential-election.jpg?w=397&h=397&f=01273440ad3df483a50a5546ffd0a938" alt="">
+          <figcaption>
+            <h4 class="text-secondary mt-3">'We Moved To Canada in 2016 After Trump Won, We Won't Be Returning''</h4>
+          </figcaption>
+        </figure>
+        <figure class="my-4">
+          <img class="w-100" src="https://d.newsweek.com/en/full/1667965/chess-womens-chess-queens-gambit.png?w=397&h=397&f=35386f73ca5a1a7a87a128f9bd61dd2c" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add the description for `alt` attribute

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1239 @@
...
+        </div>
+      </section>
+
+      <div class="row m-0">
+        <section class="subscribe bg-white p-3 mt-3 mt-md-0 col-12 col-lg-8">
+          <h2>SUBSCRIBE</h2>
+          <div class="w-100 bg-light p-4 d-md-flex justify-content-between align-items-center">
+            <figure class="d-flex justify-content-center position-relative mb-5">
+              <img src="https://d.newsweek.com/en/full/1658908/cover-november-06-11-2020.jpg?w=170&q=95&f=d5654a57e5e73c6bf7d75872dcbf38f9" alt="">
+              <img class="position-absolute top-image" src="https://d.newsweek.com/en/full/1668137/cover-november-20-2020.jpg?w=170&q=95&f=4908b956c71162b16d2cf0426b9afcbf" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add the description for `alt` attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @abongsjoel 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] MIT license link is missing in the readme file.
- [ ] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/99132503-486c1900-2638-11eb-8f4a-6b3897ab750f.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)
- [ ] Add the description for `alt` attribute (For detail see the inline comments)
- [ ] Social media icon should have the navigation link

### IN THE MAGAZINE
- [ ] Manage the spaces like the original section, as explained in image

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://abongsjoel.github.io/Newsweek-clone/)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @abongsjoel **

Great job with the project so far, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] please make sure to remove the horizontal margin in the main section.

- [ ] please make sure to use another logo png picture because the is one is pixelized. 

- [ ] please make sure to correct the issue mentioned by the previous TSE which is that the in the magazine section pictures are too small. here are some tips:

- set a fixed height and width for the images, and use the object-fit CSS attribute to make them look natural.


please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @abongsjoel  , your project is approved :tada: :tada: :tada: 
You have implemented all requested changes for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**